haiku-libs/sagebrush/sagebrush-1.0.0.recipe

```diff
@@ -0,0 +1,55 @@
+SUMMARY="An interface library for image editor"
+DESCRIPTION="SageBrush is an interface library for image editor. It provides \
+a kit to build an image editor easily without starting from scratch."
+HOMEPAGE="https://github.com/HaikuArchives/SageBrush"
+COPYRIGHT="2012 Vladislav Burundukov"
+LICENSE="MIT"
+REVISION="1"
+srcGitRev="d9b3945cb78c7070aa8b48d2e0bc6a2007fa3601"
+SOURCE_URI="https://github.com/HaikuArchives/SageBrush/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="d152e0d572a8eb0b8fc39a531cc1466827a9b96526bc5669dd181ef77ed32940"
+SOURCE_FILENAME="SageBrush-$srcGitRev.tar.gz"
```

Comment:
```diff
-SOURCE_FILENAME="SageBrush-$srcGitRev.tar.gz"
+SOURCE_FILENAME="SageBrush-$portVersion-srcGitRev.tar.gz"
```
This is highly recommended (although not strictly required.)

Reply:
It's [somewhat required][0]

> ### Archive name
>
>Our policy specifies that all archives' names should be in <port name>-<port version> format. Again, most packages are named like that. For those that are not, use SOURCE_FILENAME attribute.

Also, as this project have no release, shouldn't the port version have `~git` suffix?

[0]: https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#archive-name